matrix_functions_types.py

```diff
@@ -81,13 +81,13 @@ class CoupledHigherOrderConfig(RootInvConfig):
 
 
 @dataclass(init=False)
-class EigenvalueCorrectionConfig(PreconditionerComputationConfig):
-    """Base dataclass for matrix eigenvector method configurations in eigenvalue-corrected Shampoo."""
+class EigenvectorConfig(MatrixFunctionConfig):
+    """Base dataclass for matrix eigenvector (`matrix_eigenvectors`) method."""
 
 
 @dataclass(kw_only=True)
-class EighEigenvalueCorrectionConfig(EigenvalueCorrectionConfig):
-    """Configuration for eigendecomposition method used in eigenvalue-corrected Shampoo.
+class EighConfig(EigenvectorConfig):
```

Comment:
Can we change the name of `EighConfig` to something like `EigenBasisConfig`? Otherwise, `EighConfig` and `EigenConfig` seem like the same config.

Reply:
Addressed in #62.